engines/mxnet/mxnet-engine/build.gradle

```diff
@@ -33,13 +33,11 @@ pmdMain.source = 'src/main/java'
 tasks.register('jnarator') {
     dependsOn ":engines:mxnet:jnarator:jar"
     finalizedBy verifyJava
+    //declare inputs and outputs to allow Gradle to cache the task or skip it if inputs do not change
     inputs.file("${project.projectDir}/src/main/jna/mapping.properties")
         .withPathSensitivity(PathSensitivity.RELATIVE)
     outputs.dir "${project.buildDir}/generated-src"
 
-    //always cacheable if inputs are the same
-    outputs.cacheIf { true }
```

Comment:
What about this? Is there a reason we don't want to make this cacheable?

Reply:
Hm, unless caching of the outputs is enabled elsewhere (I couldn't find it), removing this will make this task not cacheable. Can you verify with exp2 or 3?

Reply:
Ah, I missed it because I was looking at grouping by type, but you are completely right, it becomes non-cacheable:

![image](https://github.com/benjie332/djl/assets/3155713/1eddde66-3880-4b66-9252-ebbc81a857f9)

I will readd this. I think I did the same to another task.

Reply:
In fact, with the 2 suggestions you made:

`Executed non-cacheable tasks:    186 tasks, 1m 10.688s total execution time`
->
`Executed non-cacheable tasks:    150 tasks, 1m 1.105s total execution time`